assets/queries/googleDeploymentManager/s3_bucket_without_versioning/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "227c2f58-70c6-4432-8e9a-a89c1a548cf5",
+  "queryName": "S3 Bucket Without Versioning",
+  "severity": "MEDIUM",
+  "category": "Observability",
+  "descriptionText": "S3 bucket without versioning",
```

If you agree with the suggestion, please also update in `kics/assets/queries/ansible/aws/s3_bucket_without_versioning/test/positive_expected_result.json`, `kics/assets/queries/cloudFormation/aws/s3_bucket_without_versioning/test/positive_expected_result.json`, and `kics/assets/queries/terraform/aws/s3_bucket_without_versioning/test/positive_expected_result.json`

```suggestion
  "descriptionText": "S3 bucket should have versioning enabled",
```